x-pack/plugins/security_solution/public/resolver/test_utilities/simulator/index.tsx

```diff
@@ -296,13 +317,32 @@ export class Simulator {
     return this.domNodes(`[data-test-subj="${selector}"]`);
   }
 
+  /**
+   * Given a `ReactWrapper`, returns a wrapper containing immediately following `dd` siblings.
+   * `subject` must contain just 1 element.
+   */
+  public descriptionDetails(subject: ReactWrapper): ReactWrapper {
+    // find the associated DOM nodes, then return an enzyme wrapper that only contains those.
+    const subjectNode = subject.getDOMNode();
+    let current = subjectNode.nextElementSibling;
```

Comment:
🔴 by spec, this could also be a 'dt', which would cause this to fail.

`````html
<dl>
 <dt>one term</dt>
<dt>other term</dt>
<dd>description detail for both terms</dd>
</dl>
`````

seems like you could do something similar to what you're doing below, just without the 'closest' part (since you already _know_ the dds are following siblings, right?)

Reply:
:+1 i'll look into fixin that.

Reply:
Looks like you took care of that https://github.com/elastic/kibana/pull/81077/commits/7522b682924935419144c81f265fc0b6163f251d